.github/workflows/benchmark-flow.yml

```diff
@@ -65,21 +63,22 @@ jobs:
         run: make install
       - name: Build RediSearch
         run: make build
-            COORD=${{ inputs.cluster_env }}
             PROFILE=${{ inputs.profile_env }}
             SHOW=1
       - name: Prepare automation
         run: |
           ./deps/readies/bin/getpy3
+          pip3 install --upgrade pip
+          pip3 install --upgrade requests
```

Comment:
what are those? Can we move them into the `benchmarks/requirements.txt` list?

Reply:
From @raz-mon fix. I guess we can move `requests` but upgrading `pip` should be before...